SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -272,8 +273,9 @@ void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
 	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
 	TWeakPtr<SSpatialOutputLog> SpatialOutputLog = GDKServices.GetSpatialOutputLog();
 
+	bool bSuccessfullyStartedRuntime = false;
 	RuntimeProcess->OnOutput().BindLambda([&RuntimeLogFileHandle = RuntimeLogFileHandle, &bStartingDeployment = bStartingDeployment,
-										   SpatialOutputLog](const FString& Output) {
+										   &bSuccessfullyStartedRuntime = bSuccessfullyStartedRuntime, SpatialOutputLog](const FString& Output) {
```

Comment:
A reference to a stack variable here is dangerous, could we also store it with bStartingDeployment?

Reply:
I can get down with that.